python/tests/plotting/plot_2d_test.py

```diff
@@ -239,7 +239,21 @@ def test_plot_2d_with_decreasing_edges():
 
 
 def test_plot_2d_binned_data():
-    plot(make_binned_data_array(ndim=2))
+    da = make_binned_data_array(ndim=2)
+    plot(da)
+    # Try without event-coord so implementation cannot use `histogram`
```

Comment:
Not sure I understand anymore when it calls `histogram`.
In `resampling_model.py`, I'm only seeing some choice based on the unit of the data, and not whether a coordinate is present. Can you remind me how all this works? Thanks

Reply:
Before the unit check there is a check for `if dim in array.bins.coords:`, i.e., if there is no event-coord it will not use `histogram`.

Reply:
Thanks! Misse that one... :-)